src/main/java/view/InputView.java

```diff
@@ -0,0 +1,39 @@
+package view;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private static final String INPUT_NAMES = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+    private static final String INPUT_HEIGHT = "최대 사다리 높이는 몇 개인가요?";
+    private static final String DELIMITER = ",";
```

Comment:
```suggestion
    private static final String INPUT_HEIGHT = "최대 사다리 높이는 몇 개인가요?";
    private static final String DELIMITER = ",";
    private static final String INPUT_NAMES = "참여할 사람 이름을 입력하세요. (이름은 쉼표(" + DELIMITER + ")로 구분하세요)";
```

이런식으로 표현하는것에 대해서는 어떻게 생각하시나요?
만약 이름은 구분하는 구분자가 변경됐을 때, DELIMITER 상수만 변경하고 INPUT_NAMES 를 수정하지 않는 휴먼에러를 방지할 수 있을것같아요!

Reply:
말씀 안해주셨으면 생각조차 못했을 것 같습니다. 감사합니다!